src/testdrive/src/action/kafka/ingest.rs

```diff
@@ -224,9 +191,15 @@ pub fn build_ingest(mut cmd: BuiltinCommand) -> Result<IngestAction, String> {
         None => None,
     };
     let timestamp = cmd.args.opt_parse("timestamp")?;
-    let publish = cmd.args.opt_bool("publish")?.unwrap_or(false);
     cmd.args.done()?;
 
+    if publish
```

Comment:
why this change?

fwiw my take is that the `publish` arg here makes it harder to reconstruct what a user would actually need to do in order to properly use a schema -- but I'm curious why you decided to remove it for protos here?  Just because it's confusing that the behavior differs from confluent tools in that it doesn't follow references?

Reply:
Yeah, it's just super hard to support properly for protos. `kakfa-ingest` with proto-formatted messages now takes a descriptor file and the name of a message in that descriptor file. Unfortunately that's not enough to publish the schema to the CSR: you need the input`.proto` file, not the compiled file descriptor set. But actually even _that's_ not quite right, because AFAICT the Confluent stack will reconstruct the definition of the _one_ proto message that will actually appear in the topic, rather than publishing the entire `.proto` file. The Rust protobuf library just doesn't support taking a `MessageDescriptor` and rendering it as a `.proto` file (while the Java library does). So seemed way easier to just mandate that we manually publish Protobuf schemas to the registry.